buildtest/buildsystem/base.py

```diff
@@ -431,6 +432,61 @@ def run(self):
         result = self.run_tests(testfile)
         return result
 
+    def check_returncode(self, recipe_returncode, result_returncode):
+        """ This method checks if return code from Buildspec recipe matches
+            return code from test. The return type is a bool either 'True'/'False'
+            depending if the return codes match
+
+            Parameters:
+
+            :param recipe_returncode: return code specified in recipe
+            :type recipe_returncode: int, required
+            :param result_returncode: return code from test
+            :type result_returncode: int, required
+            :return: A boolean return True/False based on conditional equality
+            :rtype: bool
+        """
+
+        if recipe_returncode == result_returncode:
```

Comment:
You can just return `recipe_returncode == result_returncode`

Reply:
got it, i fixed this locally.